pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
     "ics == 0.8.0.dev0",
     "python-dateutil >= 2.8",
     "pyyaml >= 6",
+    "requests",
```

Comment:
I would not add an extra dependency to `requests` or another HTTP library as the needs are very limited. [urllib.request](https://docs.python.org/3/library/urllib.request.html) should be more than enough for that.